src/guiapi/include/MItem_menus.hpp

```diff
@@ -187,3 +187,11 @@ class MI_EEPROM : public WI_LABEL_t {
 protected:
     virtual void click(IWindowMenu &window_menu) override;
 };
+
+class MI_QR_PRIVACY : public WI_SWITCH_OFF_ON_t {
+    constexpr static const char *const label = N_("Anonymized QR");
```

Comment:
I, as a user, would be confused, by what this means. Because if I haven't seen any BSOD yet then I wouldn't know what QR is this talking about.

Do we have space on the screen for something like „Anonymized Crash Reports“, or something similar?

Reply:
This is only about removing the hash from QR codes, BSOD has no QR codes yet.
We may change the name of this item into 4.3 final, this the best name I could get from our Content team yesterday.